src/torcontrol.cpp

```diff
@@ -292,10 +297,52 @@ static std::map<std::string,std::string> ParseTorReplyMapping(const std::string
             if (ptr == s.size()) // unexpected end of line
                 return std::map<std::string,std::string>();
```

Comment:
The loop above doesn't correctly handle the case of an *escaped* backslash just before the closing quote - e.g. (C syntax) ``"\"\\\\\""`` should represent (C syntax) ``"\\"`` and has a valid ending quote.